src/lib/openapi/spec/admin-segment-schema.test.ts

```diff
@@ -0,0 +1,59 @@
+import { validateSchema } from '../validate';
+import { AdminSegmentSchema } from './admin-segment-schema';
+
+test('updateEnvironmentSchema', () => {
```

Comment:
This was probably a copy-paste error

```suggestion
test('adminSegmentSchema', () => {
```